.gitignore

```diff
@@ -46,6 +47,9 @@ dotnet-uninstall-debian-packages.sh
 *.nupkg
 *.AppImage
 
+# Ignore certificates
+*.pfx
```

Comment:
Similarly, the pfx files should be generated outside of the repo structure

Reply:
The question is where. We don't have a cross-platform temp dir and I would very much prefer not to put it in `$HOME`. Ideally it would be on the `$TestDrive`, but anything that tries to operate across `Describe` blocks with a generated certificate will fail.

Arguably we should refactor things to do exactly that, but I think that is out of scope of this PR.

Reply:
For now, `Path::GetTempPath()` is good enough I think